ui/apps/platform/src/Components/PatternFly/CheckboxSelect.tsx

```diff
@@ -128,15 +154,13 @@ function CheckboxSelect({
                     setIsOpen(nextOpen);
                 }}
                 toggle={toggle}
-                popperProps={
-                    menuAppendTo
-                        ? {
-                              appendTo: menuAppendTo,
-                          }
-                        : undefined
-                }
+                shouldFocusToggleOnSelect
+                popperProps={{
+                    appendTo: menuAppendTo,
+                    position,
+                }}
             >
-                <SelectList>{enhancedChildren}</SelectList>
+                <SelectList style={{ minWidth: '150px' }}>{enhancedChildren}</SelectList>
```

Comment:
I should probably not use hardcoded values, now that I'm looking at this again. Will fix.